pdns/recursordist/test-syncres_cc4.cc

```diff
@@ -1478,7 +1478,7 @@ BOOST_AUTO_TEST_CASE(test_dnssec_bogus_bad_algo)
   vector<DNSRecord> ret;
   int res = sr->beginResolve(target, QType(QType::NS), QClass::IN, ret);
   BOOST_CHECK_EQUAL(res, RCode::NoError);
-  BOOST_CHECK_EQUAL(sr->getValidationState(), vState::BogusNoValidRRSIG);
+  BOOST_CHECK_EQUAL(sr->getValidationState(), vState::BogusNoValidDNSKEY);
```

Comment:
It's even weirder here, we have a RRSIG signed with a wrong algo. Or do we consider code 9 to mean "we do not know this DNSKEY"?